k8s/postgresql/schema.yaml

```diff
@@ -18,16 +18,34 @@ properties:
     title: Volume
     description: The size of Volume (Gi) in the solution
     default: 24
+  exporter.image:
+    type: string
+    default: $REGISTRY/exporter:latest
```

Comment:
We should tag this image as `$TAG` instead of latest. All republished images should have the same tag to be compatibility, so if we update the version of the application, then the exporter will be updated as well.

It seems like the readme and Makefile should be extended about this image as well.

@jprzychodzen FYI

Reply:
@wgrzelak Thanks, we've already discussed this outside GitHub, but I'm glad that you have reviewed this!